vital/util/tests/CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ kwiver_discover_gtests(vital any_converter      LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital data_stream_reader LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital string             LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital string_editor      LIBRARIES ${test_libraries})
-kwiver_discover_gtests(vital thread_pool        LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital timer              LIBRARIES ${test_libraries})
 kwiver_discover_gtests(vital token_expander     LIBRARIES ${test_libraries})
+
+if (NOT WIN32)
```

Comment:
Please consider adding a note here *why* this is being excluded on Windows.

Reply:
Good call, will do .. thanks for chiming in. Didn't think I'd get you for a while.